priv/repo/migrations/20170607142123_create_setting.exs

```diff
@@ -0,0 +1,17 @@
+defmodule CoursePlanner.Repo.Migrations.CreateSetting do
+  use Ecto.Migration
+
+  def change do
+    create table(:settings) do
```

Comment:
The advantage is that it would be easier to add and remove values because they are not coupled to the schema

The disadvantage is that the ui needs to be dynamic to add new fields, and maybe we can't validate the field type